subprojects/degrees-and-programs/source/static/app/assemble/README.md

```diff
@@ -0,0 +1,3 @@
+# brei-assemble-structure
+The BREI Assemble.io scructure
+ 
```

Comment:
This is another automatically generated file in our generator. I can delete it since there's not much here.